docs/fieldnorm/Software life cycle processes.md

```diff
@@ -0,0 +1,67 @@
+# 1. Software life cycle processes
+
+## 1.1 General
+One of the main responsibilities of bioinformaticians within a diagnostic setting involves the development and maintenance of software used in diagnostic processes. Because ISO-15189 does not cover software development, it is essential that in-house software is created and maintained in accordance with recognized harmonized standards, such as IEC-62304. The IEC-62304 description of the software development life cycle (SDLC) is consistent with the practices of most software-centric entities. 
+
+## 1.2 Software Development Life Cycle
+SDLC processes that are commonly used include planning, requirements, architectural design, development, testing, deployment, and maintenance. Laboratories should establish documented guidelines that cover all SDLC stages.
+
+## 1.2.1 Planning
+Planning entails developing a comprehensive software development plan. This includes brainstorming the initial ideas, ideation of the final goal, determining the necessary resources, creating a schedule, and establishing achievable milestones. Bioinformaticians should establish a clear communication path with stakeholders during this stage to ensure that the plan meets the goal.
+Within a laboratory, software development often occurs in the background of laboratory tests. The laboratory therefore needs to develop a solid strategy for change management for its tests that emphasizes all these features, and software development should be an integral part of this strategy.
+Smaller-scale software development and maintenance cycles that might not affect other areas of a laboratory should maintain a similar strategy on a smaller scale and be less laborious, and they might use tooling that is more effective for software development, such as code based documentation, an online documentation system that is more suitable for software development planning (like Confluence, or wiki pages), an issue management system (like issue manager or Jira) or combinations thereof.
```

Comment:
"Smaller-scale software development and maintenance cycles that might not affect other areas of a laboratory should maintain a similar strategy on a smaller scale and be less laborious, and they might use tooling that is more effective for software development, such as code based documentation, an online documentation system that is more suitable for software development planning (like Confluence, or wiki pages), an issue management system (like issue manager or Jira) or combinations thereof. " - This sentence is pretty long so it feels difficult to read to me, and the words 'laboratory' and 'laborious' in the same sentence seem similar but refer to different things (a physical lab and the amount of effort put into work), which also confuses me. I would split this sentence into multiple sentences.